server/models/intervention.ts

```diff
@@ -17,6 +17,7 @@ export default interface Intervention {
   npsRegion: NPSRegion | null
   pccRegions: PCCRegion[]
   serviceCategory: ServiceCategory
```

Comment:
Can we also remove the singular category, please? It is superseded with the list

Reply:
To expand on this, we can't set the singular category as it was inferred from the contract.

Now we need a subset of categories allowed by the contract...

Reply:
I believe Lawrence kept the singular category to make this change backwards compatible, so we don't have to update it all in one go, as we'll be writing code to switching over to the new behaviour and access multiple service categories in future stories, during which we can phase out the superseded behaviour.

Reply:
I'm not working today, but saw this and thought I should respond to avoid confusion – yes, what George said. I think that if you take a look at my commit messages, I explained this, but sorry if it's still not clear!